extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js

```diff
@@ -42,20 +42,9 @@ describe.configure().ifNewChrome().run('amp-lightbox-gallery', function() {
       win.AMP_MODE.localDev = true;
     });
 
-    it('should open and close correctly', () => {
+    it('should open and close correctly', done => {
```

Comment:
Returning the `openLightbox` Promise chains should be enough to tell Mocha that the test finished. When you use Promises, the `done` callback is almost always unnecessary

Reply:
I thought I saw some infra best practice notice about preferring `done` over returning Promises from @rsimha in some PR review that I once read, but I can't find it anymore. @rsimha do you have an opinion on return promise vs. calling `done()`?

Reply:
You could do either. If you're calling a function that returns a promise, simply return it from your test. If you need to do stuff after that, do your own promise synchronization and finally call done when the test can safely return.

Reply:
See https://mochajs.org/#asynchronous-code